### PR TITLE
argmax uses global index

### PR DESCRIPTION
**Pull request by nils-braun**
This should solve #175 by adding .value (as Max said in #175). I have also added a test to not let this happen again.

### CODE REVIEW

**Comment by MaxBenChrist** [comment]
haha Nils, the man who submits pr faster than his shadow. ;)

There are also another few occurrences of argmax, we should check them. Further 

https://github.com/blue-yonder/tsfresh/blob/master/tests/feature_extraction/test_extraction.py#L114-LL16

**Comment by nils-braun** [comment]
@MaxBenChrist In the other two cases of calling np.argmax, we use np.asarray before, which should have more or less the same effect as using `.value`.

**Comment by MaxBenChrist** [comment]
> @MaxBenChrist In the other two cases of calling np.argmax, we use np.asarray before, which should have more or less the same effect as using .value.

You are right. 

Can you add another drop index to https://github.com/blue-yonder/tsfresh/blob/master/tests/feature_extraction/test_extraction.py#L98 ?

File: tests/feature_extraction/test_feature_calculations.py [hunk 1]
```diff
@@ -320,6 +320,16 @@ def test_mass_quantile(self):
         six.assertCountEqual(self, list(res.index), expected_index)
         self.assertAlmostEqual(res["TEST__index_mass_quantile__q_0.5"], 0.5, places=1)
 
+        # Test for parts of pandas series
```
Comment (MaxBenChrist) [inline tests/feature_extraction/test_feature_calculations.py]: ?

File: tests/transformers/test_full_pipeline.py [hunk 2]
```diff
@@ -37,14 +37,6 @@ def test_relevant_extraction(self):
                                   'a__autocorrelation__lag_1',
                                   'a__binned_entropy__max_bins_10',
                                   'a__count_above_mean',
-                                  'a__index_mass_quantile__q_0.1',
```
Comment (MaxBenChrist) [inline tests/transformers/test_full_pipeline.py]: Why did you remove them?

File: tsfresh/feature_extraction/feature_calculators.py [hunk 3]
```diff
@@ -788,11 +788,13 @@ def index_mass_quantile(x, c, param):
...
     res = pd.Series()
 
+    x = pd.Series(x)
+
     if s == 0: # all values in x are zero or it has length 0
         for config in param:
             res["{}__index_mass_quantile__q_{}".format(c, config["q"])] = np.NaN
     else: # at least one value is not zero
-        mass_centralized = np.cumsum(np.abs(x)) * 1.0 / s
+        mass_centralized = (np.cumsum(np.abs(x)) * 1.0 / s).values
```
Comment (MaxBenChrist) [inline tsfresh/feature_extraction/feature_calculators.py]: I would add the `.value`to the x inside the abs call. I think it is more readable

File: tests/transformers/test_full_pipeline.py (hunk 2 above)
Comment (nils-braun) [inline tests/transformers/test_full_pipeline.py]: As I have said in the it message, they are not relevant anymore

File: tests/feature_extraction/test_feature_calculations.py (hunk 1 above)
Comment (nils-braun) [inline tests/feature_extraction/test_feature_calculations.py]: Well, the issue was not spotted because we calculate the feature always on the full series

Comment (MaxBenChrist) [inline tests/feature_extraction/test_feature_calculations.py]: Why do you need a vector with over 150 values? are 3 not enough?

Comment (nils-braun) [inline tests/feature_extraction/test_feature_calculations.py]: I just copied the example from above and added some values. It does not matter how much.

Comment (MaxBenChrist) [inline tests/feature_extraction/test_feature_calculations.py]: I was thinking about keeping the runtime of the tests low
